src/Uno.UI/UI/Xaml/FrameworkElementHelper.cs

```diff
@@ -37,7 +37,7 @@ public static void SetDataTemplateRenderPhases(FrameworkElement target, int[] de
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public static bool IsUiAutomationMappingEnabled { get; set; } = false;
 
-		public static void SetBaseUri(IFrameworkElement target, string uri)
+		public static void SetBaseUri(FrameworkElement target, string uri)
```

Comment:
Codacy found an issue: [Either change this parameter type to 'System.Uri' or provide an overload which takes a 'System.Uri' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5554394)